### PR TITLE
contact page style update

### PR DESCRIPTION
**Pull request by eakholod**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h1 class="about-title">About</h1>
       <a href="https://www.facebook.com/darya.tolok.750" class="fa fa-facebook"></a>
       </div>
       <p>
-        Photo: Anton Tolok & Tim Tolok
+        Photo: Anton Tolok, Tim Tolok, Elizabeth Popova
```
Comment (jacobvr) [inline about.html]: I would suggest for the last person use `&` instead of `,`, what do you think?

File: contact.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
       <a href="https://www.facebook.com/darya.tolok.750" class="fa fa-facebook"></a>
       </div>
       <p>
-        Photo: Anton Tolok & Tim Tolok
+        Photo: Anton Tolok, Tim Tolok, Elizabeth Popova
```
Comment (jacobvr) [inline contact.html]: Same as above

File: index.html [hunk 3]
```diff
@@ -56,7 +56,7 @@ <h3>Music Theory</h3>
   <a href="https://www.facebook.com/darya.tolok.750" class="fa fa-facebook"></a>
   </div>
   <p>
-    Photo: Anton Tolok & Tim Tolok
+    Photo: Anton Tolok, Tim Tolok, Elizabeth Popova
```
Comment (jacobvr) [inline index.html]: Same

File: styles.css [hunk 4]
```diff
@@ -17,7 +17,6 @@ time, mark, audio, video {
 }
 
 body{
-background-color: white;
```
Comment (jacobvr) [inline styles.css:20]: What is the reason for removing this? what if some browser has a default background? Do you want to have a safe-guard here and have a default background color?

File: styles.css [hunk 5]
```diff
@@ -181,6 +180,7 @@ a {
     grid-template-rows: repeat(1, 35vw);
     grid-gap: 15px;
     margin: 0px 15px 15px;
+
```
Comment (jacobvr) [inline styles.css]: Extra space?

File: styles.css [hunk 6]
```diff
@@ -230,7 +230,10 @@ a {
 
 .about-content {
   margin-bottom:20px ;
+  margin-top: 20px;
   flex: 1 1 50%;
+  /* background-color: rgb(240, 240, 240); */
```
Comment (jacobvr) [inline styles.css]: do you need this or could be deleted?

**Review by jacobvr — COMMENTED** [review]
I have a few questions below

**Review by jacobvr — CHANGES_REQUESTED** [review]
Photo credits look like a link and it's confusing, please remove underline on hover

Everywhere you have Darya's email please use a special email link:

```
<a href="mailto:darya.vo916@gmail.com">darya.vo916@gmail.com</a>
```

**Comment by jacobvr** [comment]
Also, please remove underline effect for YT and FB

File: about.html (hunk 1 above)
Comment (eakholod) [reply about.html]: Yes, that is a great catch.

File: styles.css (hunk 4 above)
Comment (eakholod) [reply styles.css:20]: I was messing around with styles. But speaking of browser default background, I would leave it as is. I think it very small issue which could happen very likely.

**Review by jacobvr — APPROVED** [review]
LGTM, good work

**Pull request merged**